src/tools/compiletest/src/header.rs

```diff
@@ -564,6 +566,13 @@ impl Config {
         }
     }
 
+    fn parse_cfg_prefix(&self, line: &str, prefix: &str) -> bool {
```

Comment:
Can you rename this to something like `has_cfg_prefix`? It's not really parsing right now.

Reply:
Sure. Can you point what's wrong with parsing? Thanks.

Reply:
There's nothing wrong, just that a function returning `bool` seems like it shouldn't be called a `parse_*` function since it's not actually returning the results of the parsing, so to speak. I don't feel too strongly about this but `has_cfg_prefix` seems clearer to me.

Reply:
I misunderstood, I will update the function name.